addons/account/models/account_analytic_line.py

```diff
@@ -26,6 +30,43 @@ def _check_company_consistency(self):
         if self._cr.fetchone():
             raise UserError(_("You can't set a different company on your analytic account since there are some journal items linked to it."))
 
+    @api.depends('account_move_line_ids')
+    def _compute_account_move_count(self):
+        for account in self:
+            account.invoice_count = len(account.account_move_line_ids.move_id.filtered(lambda m: m.move_type == 'out_invoice' or m.move_type == 'out_refund'))
+            account.vendor_bill_count = len(account.account_move_line_ids.move_id.filtered(lambda m: m.move_type == 'in_invoice' or m.move_type == 'in_refund'))
+
+    def action_view_invoice(self):
+        self.ensure_one()
+        invoices = self.account_move_line_ids.move_id.filtered(lambda m: m.move_type == 'out_invoice' or m.move_type == 'out_refund')
```

Comment:
Is it not possible to provide a domain instead of a list of ids? The list of ids might be huge.